code/player/Inventory.cs

```diff
@@ -42,13 +42,13 @@ public override bool Add(Entity entity, bool makeActive = false)
         {
             TTTPlayer player = Owner as TTTPlayer;
 
-            if (IsCarryingType(entity.GetType()))
-            {
-                return false;
-            }
```

Comment:
Shouldn't this be kept if you trying to pickup something you already have that is not derived from `ICarriableItem`?

Reply:
Damn, good catch. I will revert.

Reply:
Reverted